### DISCUSSION TITLE
Vote for Lightdash to support your favourite warehouse

### DISCUSSION THREAD

**Discussion by owlas** [lightdash/lightdash#1112] [Ideas]
Hey all!

If Lightdash doesn't support your favourite database / data warehouse you can vote for it here.

**How to vote**: search the comments for the database name and vote on the comment. If the comment doesn't exist add a new one.

**Comment by owlas** [comment]
[Rockset](https://rockset.com)

**Comment by owlas** [comment]
[AWS Athena](https://aws.amazon.com/athena/)

**Reply by djayatillake** [reply]
I can't see many people wanting to use this as a DWH it's more or less defunct at Lyst now. Everyone who uses it hates it.

**Reply by owlas** [reply]
Athena was a community vote I've migrated here from issues. 

Instead of a thumbs down we can prioritise based on those with the most upvotes, if this sees little love then it won't be built soon!

**Comment by djayatillake** [comment]
[Firebolt](https://www.firebolt.io/?utm_term=firebolt&utm_campaign=PPC-Google-Brand&utm_source=google&utm_medium=cpc&utm_content=559922630869&hsa_acc=1333801409&hsa_cam=12067789830&hsa_grp=134807778132&hsa_ad=559922630869&hsa_src=g&hsa_tgt=kwd-436108302&hsa_kw=firebolt&hsa_mt=e&hsa_net=adwords&hsa_ver=3&cq_med=&cq_plac=&cq_net=g&cq_plt=gp&gclid=CjwKCAiAz--OBhBIEiwAG1rIOp7e_NG0xxgw9zwktRYWiLfohoZEna1FPHybCuICphc28vTHJE8GlRoCq1IQAvD_BwE)

**Comment by djayatillake** [comment]
[Clickhouse](https://clickhouse.com/)

**Reply by tobiascadee** [reply]
Are there any plans for this?

**Reply by jpetey75** [reply]
Hi @tobiascadee we don't have plans to add in the short term, but I believe there are a few people [connecting to Clickhouse via Trino](https://trino.io/docs/current/connector/clickhouse.html) since Lightdash does work with Trino.

**Reply by tobiascadee** [reply]
okay thanks!

**Reply by adri** [reply]
Why ask for the users most favorite warehouses and not building native support for the most liked? :) @jpetey75

**Reply by 12ian34** [reply]
Hey @adri have you tried connecting to Clickhouse via Trino?

**Reply by adri** [reply]
Hey @12ian34 thanks, but for our use-case, hosting Trino is quite complex just to be able to connect Ligthdash. Managed services we found were also quite pricey.

Good to know or others: If you're a paying customer, it can be worth to reach out to Ligthdash customer support. I wasn't aware, but it can make a difference in prioritizing (make sense, but I didn't realize it first).

**Reply by hamzahc1** [reply]
Update for folks - we've just opened a PR to add Clickhouse support https://github.com/lightdash/lightdash/pull/16988

**Reply by hamzahc1** [reply]
This is now publicly available: https://docs.lightdash.com/get-started/setup-lightdash/connect-project#clickhouse

**Comment by djayatillake** [comment]
[Apache Pinot](https://pinot.apache.org/) / [Startree](https://www.startree.ai/)

**Reply by dvyach** [reply]
any work around to try to have pinot connect to lightdash?

**Reply by 12ian34** [reply]
@dvyach you can try the [Trino Pinot Connector](https://trino.io/docs/current/connector/pinot.html) in the meantime!

**Comment by owlas** [comment]
MySQL

**Comment by tokestermw** [comment]
[Trino / PrestoSQL](https://trino.io)

**Comment by zsombor-flds** [comment]
DuckDB

**Comment by eranhz** [comment]
What are your thoughts around adding Spark support?

**Comment by chllming** [comment]
Motherduck

**Comment by tlinkin** [comment]
Redshift with IAM for Glue / Lake Formation

#14957
